### PR TITLE
Update wiki description of interaction framework

### PR DESCRIPTION
**Pull request by Schwaggot**
**When merged this pull request will:**
- update the wiki description of the interaction menu framework
- adds a hint that functions will be not executed on server
- removes 'it is generally preferred to add actions via config' as suggested by jonpas on Slack

### CODE REVIEW

**Comment by Dystopian** [comment]
For Zeus there is `ace_interact_menu_fnc_addActionToZeus`. Also functions **will** be executed on server if it's not dedicated (server with interface).

**Comment by jonpas** [comment]
I'd reword it to say they are all local and entire menu is client-side. Because if you add it on server, it'll be on server, you just can't use it.

**Comment by Schwaggot** [comment]
I've updated it. My point was that the function actually exit if ```!hasInterface```, e.g., https://github.com/acemod/ACE3/blob/master/addons/interact_menu/functions/fnc_createAction.sqf#L31

**Comment by jonpas** [comment]
Ooooh, in that case I prefer the first version. :grimacing:

**Comment by TheMagnetar** [comment]
@Schwaggot can you modify it again so we can merge it?

**Comment by Schwaggot** [comment]
@TheMagnetar sure, I will do so either later today or tomorrow.
